examples/chat/client/tasks.nim

```diff
@@ -210,35 +290,44 @@ proc login*(account: int, password: string) {.task(kind=no_rts, stoppable=false)
   let allAccounts = status.getPublicAccounts()
 
   var
-    event: LoginResult
+    event: events.LoginResult
```

Comment:
Noticed the `events` prefix - does it make it sense to enforce name qualification via `from ./events import nil` instead of `import ./events`?

Reply:
What would be the impact on the end result?

Reply:
Possibly preventing name collisions in the future.

Reply:
Yeah, i agree that using the suffix `Result` for events is a bit misleading/confusing since we are now heavily using the `nim-result` lib which uses types that end in `Result`.

Reply:
All `Result` suffixes for event types have been replaced with `Event`.